venom_core/services/runtime_switch_service.py

```
@@ -47,6 +47,28 @@ async def _release_ollama_model(endpoint: str, model_name: str) -> bool:
 
+async def _release_multi_runtime_models(endpoint: str) -> bool:
+    """Ask multi_runtime daemon to unload all models before service stop."""
+    if not endpoint:
+        return False
+    base = endpoint.rstrip("/").removesuffix("/v1")
+    unload_url = f"{base}/v1/daemon/unload"
+    try:
+        async with httpx.AsyncClient(timeout=_CLEANUP_TIMEOUT) as client:
+            response = await client.post(unload_url)
+        if response.status_code >= 400:
+            logger.warning(
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)